api/maven-api-core/src/main/java/org/apache/maven/api/ArtifactCoordinate.java

```diff
@@ -23,64 +23,70 @@
 import org.apache.maven.api.annotations.Nonnull;
 
 /**
- * The {@code Coordinate} object is used to point to an {@link Artifact}
- * but the version may be specified as a range instead of an exact version.
+ * Partial identification of an {@code Artifact} in a Maven repository.
+ * Each {@code ArtifactCoordinate} instance is basically a pointer to a file in the Maven repository,
+ * except that the exact version may not be known yet.
  *
  * @since 4.0.0
  */
 @Experimental
 @Immutable
 public interface ArtifactCoordinate {
```

Comment:
I'm just curious -- why isn't the name of the interface plural -- `ArtifactCoordinates` to begin with? I mean -- the interface isn't providing one single coordinate. It's providing an artifact's coordinates.